lib/x11test.pm

```diff
@@ -641,7 +641,8 @@ sub firefox_check_popups {
 }
 
 sub firefox_open_url {
-    my ($self, $url) = @_;
+    my ($self, $url, $do_not_check_loaded_url) = @_;
```

Comment:
You call this as `$self->firefox_open_url('http://www.url.com/', do_not_check_loaded_url => 1);` which is 3 parameters. But you parse just 2 params.

Maybe you wanted
```perl
  my ($self, $url, %args) = @_;
  if ($args{do_not_check_loaded_url}) {...}
```

Reply:
why ?